packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -150,9 +156,21 @@ export class PluginLedgerConnectorBesu
       `${fnTag} options.prometheusExporter`,
     );
 
+    // const web3Provider = new Web3.providers.HttpProvider(
```

Comment:
@travis-payne If the commented code is no longer useful then I recommend to just remove it completely.

Reply:
@travis-payne What was the reason behind removing the HTTP provider? (IIRC we had both WS and HTTP available - might be wrong though)

Reply:
This bit of code isn't even in main branch, we only use the HTTP provider to sign a transaction (line 717 on main). I think I was just testing something. Will remove.